### PR TITLE
chore: Added issue creation template, pull request template, and automatic assignment of reviewers to PRs

### PR DESCRIPTION
**Pull request by segunjkf**
This PR introduces GitHub issue and PR templates for use when creating issues or PRs, along with a method to automatically assign reviewers to PRs.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# they will be requested for review when someone opens a pull request.
+*	@haroldsphinx @segunjkf @calvin-puram @voltagebots
```
Comment (haroldsphinx) [inline .github/CODEOWNERS:4]: Lets create a Github Group and add every under..its easier to manage that way

**Review by haroldsphinx — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (segunjkf) [reply .github/CODEOWNERS:4]: @haroldsphinx makes sense, but i don't know if i have enough permission to do that.

Comment (voltagebots) [reply .github/CODEOWNERS:4]: There is a @blockopsnetwork/devops  group

Comment (segunjkf) [reply .github/CODEOWNERS]: Done. Thanks @voltagebots

**Review by haroldsphinx — APPROVED** [review]

**Pull request merged**